app/views/HasNirmsView.scala.html

```diff
@@ -19,12 +19,23 @@
     formHelper: FormWithCSRF,
     govukErrorSummary: GovukErrorSummary,
     govukRadios: GovukRadios,
-    govukButton: GovukButton
+    govukButton: GovukButton,
+    link: components.Link
 )
 
 @(form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
 
 @layout(pageTitle = title(form, messages("hasNirms.title"))) {
+    <h1 class="govuk-heading-l">@messages("hasNirms.h1")</h1>
+
+    <p class="govuk-body">@messages("hasNirms.p1")</p>
+
+    <p class="govuk-body">
+        @Html(messages("hasNirms.p2", link(messages("hasNirms.p2.linkText"),
```

Comment:
do we want to be using the link component? I think this was the one Nick specifically warned against